src/neuroconv/datainterfaces/behavior/fictrac/fictracdatainterface.py

```diff
@@ -72,17 +81,12 @@ def __init__(
 
     def get_metadata(self):
         metadata = super().get_metadata()
-        from datetime import datetime
 
-        config_file = self.file_path.parent / "fictrac_config.txt"
-        if config_file.exists():
-            self._config_file = parse_fictrac_config(config_file)
-            date_string = self._config_file["build_date"]
-            date_object = datetime.strptime(date_string, "%b %d %Y")
-
-            metadata["NWBFile"].update(
-                session_start_time=date_object,
-            )
+        try:
+            session_start_time = extract_session_start_time(self.file_path)
+            metadata["NWBFile"].update(session_start_time=session_start_time)
+        except:
```

Comment:
I always try to avoid bare excepts; is there a specific error that is known to occur here? Is there no other way to determine the condition using `if/else`?